API/device/artik053.py

```diff
@@ -41,7 +41,7 @@ def get_test_path(self):
         '''
         Return the test path on the device.
         '''
-        return '/rom/test'
+        return '/rom'
```

Comment:
Was the path incorrect in the previous patch, right? As I checked all the tests are copied to a `res` folder that is mounted as `/rom`.

Reply:
No. The previous patch is correct. I just changed the copy location to match the test location of iot.js and jerryscript in the target.

Before
```
iot.js : /rom/test
jerryscript: /rom/jerry
```

After
```
iot.js : /rom
jerryscript: /rom
```

Reply:
I see.